sites/public/lib/ApplicationConductor.ts

```diff
@@ -205,13 +205,12 @@ export default class ApplicationConductor {
   }
 
   routeTo(url: string) {
-    void Router.push(url).then(() => window.scrollTo(0, 0))
```

Comment:
What was the point of the scrollTo logic before?

Reply:
It was going to a new page but not scrolling up to the top of the page by default. But in Next v10 they added that functionality automatically (sort of bonkers it wasn't there from the get-go).